templatestore/frontend/src/components/Dropdowncomp.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styles from '../style/templateScreen.less';
+export default function Dropdowncomp(props) {
+    return (
+        <div className={styles.teAttributesRow}>
```

Comment:
Can we have better names for styles? `te` is not making much sense.

Reply:
This style scheme is used across the project, i just used them.

Reply:
Seems like these are predefined, you can skip this comment.